cli/flags.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+use clap::{App, AppSettings, Arg, ArgMatches};
+use crate::ansi;
 use deno::v8_set_flags;
-use getopts;
-use getopts::Options;
```

Comment:
getopts should be removed from Cargo.toml and cli/BUILD.gn too

Reply:
Will do